internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -38,25 +38,26 @@ import (
 )
 
 const (
-	observabilityNamespace          = constants.ObservabilityOperatorNamespace
-	observabilityOperatorGroupName  = "observability-operator-group-name"
-	observabilityCatalogSourceName  = "observability-operator-manifests"
-	observabilitySubscriptionName   = "observability-operator"
-	observatoriumDexSecretName      = "observatorium-configuration-dex"
-	observatoriumSSOSecretName      = "observatorium-configuration-red-hat-sso"
-	syncsetName                     = "ext-managedservice-cluster-mgr"
-	strimziAddonNamespace           = constants.StrimziOperatorNamespace
-	strimziQEAddonNamespace         = "redhat-managed-kafka-operator-qe"
-	kasFleetshardAddonNamespace     = constants.KASFleetShardOperatorNamespace
-	kasFleetshardQEAddonNamespace   = "redhat-kas-fleetshard-operator-qe"
-	openIDIdentityProviderName      = "Kafka_SRE"
-	mkReadOnlyGroupName             = "mk-readonly-access"
-	mkSREGroupName                  = "kafka-sre"
-	mkReadOnlyRoleBindingName       = "mk-dedicated-readers"
-	mkSRERoleBindingName            = "kafka-sre-cluster-admin"
-	dedicatedReadersRoleBindingName = "dedicated-readers"
-	clusterAdminRoleName            = "cluster-admin"
-	kafkaInstanceProfileType        = "bf2.org/kafkaInstanceProfileType"
+	observabilityNamespace                     = constants.ObservabilityOperatorNamespace
+	observabilityOperatorGroupName             = "observability-operator-group-name"
+	observabilityCatalogSourceName             = "observability-operator-manifests"
+	observabilitySubscriptionName              = "observability-operator"
+	observatoriumDexSecretName                 = "observatorium-configuration-dex"
+	observatoriumSSOSecretName                 = "observatorium-configuration-red-hat-sso"
+	syncsetName                                = "ext-managedservice-cluster-mgr"
+	strimziAddonNamespace                      = constants.StrimziOperatorNamespace
+	strimziQEAddonNamespace                    = "redhat-managed-kafka-operator-qe"
+	kasFleetshardAddonNamespace                = constants.KASFleetShardOperatorNamespace
+	kasFleetshardQEAddonNamespace              = "redhat-kas-fleetshard-operator-qe"
+	openIDIdentityProviderName                 = "Kafka_SRE"
+	mkReadOnlyGroupName                        = "mk-readonly-access"
+	mkSREGroupName                             = "kafka-sre"
+	mkReadOnlyRoleBindingName                  = "mk-dedicated-readers"
+	mkSRERoleBindingName                       = "kafka-sre-cluster-admin"
+	dedicatedReadersRoleBindingName            = "dedicated-readers"
+	clusterAdminRoleName                       = "cluster-admin"
+	kafkaInstanceProfileType                   = "bf2.org/kafkaInstanceProfileType"
+	observabilityProxyServiceAccountSecretName = "observability-proxy-credentials"
```

Comment:
Is the `observabilityProxyServiceAccountSecretName` value going to be used outside of the  `buildDataplaneServiceaccountSecretResource` method? If not, I'd prefer having it inside the method. The scope that should be located is the minimal scope where it will be needed